supernovacontroller/sequential/i3c.py

```diff
@@ -1410,7 +1412,7 @@ def ccc_broadcast_setxtime(self, timing_parameter):
         return self._process_response("ccc_broadcast_setxtime", responses)
 
     def ccc_unicast_setxtime(self, target_address):
-        pass
+        pass # TODO see issue BMC2-1662
```

Comment:
Ok. According our offline discussion, this will be addressed for a future a release of SC.